src/services/ExplorerService/fetcher.ts

```diff
@@ -902,3 +909,10 @@ export interface TransferListRes {
 export type DASAccount = string
 
 export type DASAccountMap = Record<string, DASAccount | null>
+
+export type UdtQueryResult = {
```

Comment:
Proper nouns or abbreviations should be all uppercase or all lowercase, so here it should be `UDTQueryResult`.